Project.toml

```diff
@@ -28,25 +29,16 @@ CUDAExt = "CUDA"
 CairoMakieExt = "CairoMakie"
 
 [compat]
-CUDA = "3.8.4, 3.12, 4.4"
-CairoMakie = "0.7, 0.10.7, 0.11, 0.12"
+CUDA = "5"
+CairoMakie = "0.7, 0.10.7, 0.11, 0.12, 0.13"
 CpuId = "0.3"
 DocStringExtensions = "0.9"
 Glob = "1.3"
-HDF5 = "0.16"
+HDF5 = "0.16, 0.17"
 NVTX = "0.3"
 Reexport = "1.2"
-TestItemRunner = "0.2"
-ThreadPinning = "0.3, 0.4, 0.5, 0.6, 0.7"
+TestItemRunner = "0.2, 1"
```

Comment:
```suggestion
```
You want to move this compat to the `test` environment, but with this version I get
```
ERROR: Compat `TestItemRunner` not listed in `deps`, `weakdeps` or `extras` section at "/tmp/jl_S0EMgC/Project.toml".
```
which makes it impossible to install your version.